### PR TITLE
Feature/dashboard skeleton #24

### LINKED ISSUE #24: Implement dashboard skeleton
- [x] Integrate vaccination dashboard (iframe) to project monitoring

In terms of charts and tables, for now, we will start with a simple dashboard per dataset
- [ ] Target vs. actual comparison (per month)
- [ ] Total target vs. total actual comparison  (cumulative)
- [ ] Table with as much information as possible
- [ ] Table with gender breakdown for people indicator
- [ ] Pie charts entered/validata data => We need to work on a validation workflow and document it.

### PR DESCRIPTION
**Pull request by laurapareja**
### :pushpin: References

* **Issue:** Closes #24 
### :memo: Implementation
- Implement dashboard skeleton
- Implement data-entry dashboard & remove `leftBar`
- Remove `header`
- Remove double scroll

To Implement: 
- [ ] Data Entry is disabled if there isn't target values

### :art: Screenshots
image

--- Data Entry Iframe

image

image

### CODE REVIEW

File: src/pages/dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -17,9 +17,49 @@ function getConfig() {
 }
 
 const Dashboard: React.FC = () => {
+    const [loading, setLoading] = useState(false);
     const history = useHistory();
     const goToLandingPage = () => goTo(history, "/");
     const config = getConfig();
+    const iFrameSrc = "http://localhost:8081/dhis-web-dashboard/#/nghVC4wtyzi";
```
Comment (tokland) [inline src/pages/dashboard/Dashboard.tsx]: I could not access this URL. Change to baseUrl from app-runtime (like App.jsx)

File: src/pages/dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -17,9 +17,49 @@ function getConfig() {
...
+    const iframeRef: React.RefObject<HTMLIFrameElement> = React.createRef();
+
+    const waitforElementToLoad = (iframeDocument: any, selector: string) => {
```
Comment (tokland) [inline src/pages/dashboard/Dashboard.tsx]: Minor: To lighten the main function, we can move this helpers to the top level.

**Comment by tokland** [comment]
I forgot, to easily develop pages with iframes, I do this:

1) Run chrome with ----disable-web-security flag.
2) Install this extension: https://chrome.google.com/webstore/detail/ignore-x-frame-headers/gleekbfjekiniecknbkamfmkohkpodhe

**Comment by adrianq** [comment]
Pending tasks captured in another issue -> https://github.com/EyeSeeTea/project-monitoring-app/issues/42

**Pull request merged**